File: app/src/main/java/rk/information/news/allNews/LoadingViewHolder.java
package rk.information.news.allNews;

import android.view.View;
import android.widget.ProgressBar;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import rk.information.news.R;

class LoadingViewHolder extends RecyclerView.ViewHolder {

    ProgressBar progressBar;

    public LoadingViewHolder(@NonNull View itemView) {
        super(itemView);
        progressBar = itemView.findViewById(R.id.progressBar);
    }
}

File: app/src/main/java/rk/information/news/allNews/LoadingViewHolder.kt
package rk.information.news.allNews

import android.view.View
import android.widget.ProgressBar
import androidx.recyclerview.widget.RecyclerView
import rk.information.news.R

internal class LoadingViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    var progressBar: ProgressBar = itemView.findViewById(R.id.progressBar)

}